indico/modules/events/surveys/util.py

```diff
@@ -59,3 +65,35 @@ def was_survey_submitted(survey):
     if submission_id is None:
         return False
     return bool(SurveySubmission.find(id=submission_id).count())
+
+
+class CSVSubmissionsExporter(object):
+
+    BASE_FIELDNAMES = {'submitter', 'submission_date'}
+
+    def __init__(self, survey):
+        self._survey = survey
+        self._fieldnames = self.BASE_FIELDNAMES | {question.title for question in self._survey.questions
+                                                   if not isinstance(question.field, StaticTextField)}
+
+    def generate(self):
+        with NamedTemporaryFile(suffix='indico.tmp', dir=Config.getInstance().getTempDir(), delete=False) as temp_file:
+            writer = csv.DictWriter(temp_file, fieldnames=self._fieldnames)
```

Comment:
the csv writer  has some option that it should be excel-friendly/compatible. i think we should set that